### DISCUSSION TITLE
Display more granular offer expiration

### DISCUSSION THREAD

**Discussion by danieljperry** [Chia-Network/chia-blockchain-gui#2378] [Ideas]
The default setting for expiring offers is 7 days. When an offer's expiration time is displayed in the GUI, it only lists `days` when more than 1 day remains. A default offer's expiration time is thus very quickly listed as `6 days`, even though something like `6 days, 23 hours, 59 minutes` would be more accurate. This request is to show the expiration times more like the latter.

https://github.com/Chia-Network/chia-blockchain/assets/69756004/c670ca43-eb0b-473d-9c30-bce0d0f26611

**Comment by BrandtH22** [comment]
@danieljperry , note I transferred this enhancement request to the GUI repo for better visibility 👍
